pubspec.yaml

```diff
@@ -65,7 +66,12 @@ dependencies:
   web_socket_channel: ^2.2.0
   webfeed: ^0.7.0
 
-dependency_overrides: null
+dependency_overrides:
+  firebase_analytics_web:
+    git: 
+      url: https://github.com/durduman/flutterfire.git
+      path: packages/firebase_analytics/firebase_analytics_web
+      ref: firebase_analytics_web-0.4.0+16-fix-for-web
```

Comment:
Would you mind detailing why this override is needed ?

Reply:
I've written the whole explanation in the description.
TLDR:
Versioning issue that did not support web properly. Newer versions conlfict with web3, and have compile problem, older version have a runtime crash, and the fix was introduced on a analytics_web version that is not compatible with our current dependencies.